--- E:\LabCode\RRIQA_patch\regression\HTReg_Net.py
+++ E:\LabCode\RRIQA_patch\regression\HTReg_Net.py
@@ -11,9 +11,9 @@
 
         )
 
-    def forward(self, hfc_512, tfc_512):
+    def forward(self, hfc_5112, tfc_512):
         # print(hfc_5112.shape, tfc_512.shape)
-        ht_fc = torch.cat([hfc_512, tfc_512], dim=-1)
+        ht_fc = torch.cat([hfc_5112,tfc_512],dim=-1)
         # print(ht_fc.shape)
         prd_score = self.fc(ht_fc)
 